spec/child-apps/bootstrap-rejects/bootstrap-rejects.spec.js

```diff
@@ -1,18 +1,16 @@
-const appLocation = "/base/spec/child-apps/bootstrap-rejects/bootstrap-rejects.app.js";
-
 export default function() {
 	describe(`bootstrap-rejects`, () => {
 		let childApp;
 
 		beforeAll(() => {
-			singleSpa.declareChildApplication(appLocation, location => location.hash === "#bootstrap-rejects");
+			singleSpa.declareChildApplication('./bootstrap-rejects.app.js', () => System.import('./bootstrap-rejects.app.js'), location => location.hash === "#bootstrap-rejects");
```

Comment:
I changed over all of the tests to use the new version of the declareChildApplication API so that the tests could be run in a webpack world, too.

The other thing I had to do was put literal strings into the `System.import`s, since webpack statically analyzes the code at build time and therefore isn't cool with System.importing variable-named things.

This accounts for the majority of the changes in this pr, I will point out below where the actual changes to the source code are.